### ISSUE TITLE
Drag'n'Drop audio files from a file manager

### ISSUE THREAD

**Issue by unfa** [openAVproductions/openAV-Fabla2#66]
The idea is to make it possible to drag'n'drop files from file manager onto the pads to load the samples.

I wonder if this is even possible with LV2 plugin architecture.

**Comment by harryhaaren** [comment]
Yes this is possible - the UI toolkit used for the plugin UI is the part that will "accept" the drag-n-drop contents, and as such it can send an LV2 Atom to the DSP part to load the file. Its not particularly easy, but possible yes.
